src/main/java/com/oltpbenchmark/benchmarks/featurebench/FeatureBenchWorker.java

```diff
@@ -80,7 +80,14 @@ protected void initialize() {
         FileUtil.makeDirIfNotExists(outputDirectory + "/" + explainDir);
         String fileForExplain = explainDir + "/" + workloadName + "_" + TimeUtil.getCurrentTimeString() + ".json";
         PrintStream ps;
-        String explain = "explain (analyze,verbose,costs,buffers) ";
+        String explain = "";
+        if (this.getWorkloadConfiguration().getXmlConfig().containsKey("explain_use_dist")
+            && this.getWorkloadConfiguration().getXmlConfig().getBoolean("explain_use_dist")) {
+            explain = "explain (analyze, dist) ";
```

Comment:
can you check if this works with : 
explain = "explain (analyze, verbose, costs, buffers, dist) ";